<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*学校LOGO*/
QLabel#label_logo,QLabel#label_logo_2,QLabel#label_logo_7
{
	border-image: url(:/images/logo.png);
}
/*登录模块的界面图片*/
QWidget#page_login,QWidget#page_register,QWidget#page_server
{
	border-image: url(:/images/logon-bg.jpg);
}
/*正常按钮的状态*/
QPushButton#btn_login,QPushButton#btn_cancle,QPushButton#btn_register,QPushButton#btn_rcancle,QPushButton#btn_tologin,QPushButton#btn_save_server,QPushButton#btn_scancle,QPushButton#btn_sreturn
{
	border-radius: 10px;
	border-style: solid;
	border-width: 1px;
	background-color: rgb(210, 255, 238);
}

/*文本框样式*/
QLineEdit
{
	border-radius: 5px;
}
/*标签与小按钮的字体高亮*/
QLabel#label_lusername, QLabel#label_lpwd, QPushButton#btn_toregister,QPushButton#btn_to_setserver,QLabel#label_ip,QLabel#label_port
{
	color: rgb(210, 255, 238);
}

/*按钮按下*/
QPushButton:pressed
{
	border-color: gray;
	border-width:2px;
	background-color: rgb(170, 255, 127);
}

QRadioButton
{
	color: rgb(170, 255, 255);
}
QRadioButton::indicator:checked
{
	background-color: rgb(55, 55, 255);
	border-radius: 6px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_login">
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>230</y>
         <width>321</width>
         <height>50</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_lpwd">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>密码</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edit_lpwd">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
          <property name="text">
           <string>666666</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="btn_login">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>340</y>
         <width>110</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>登录</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_cancle">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>340</y>
         <width>110</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_toregister">
       <property name="geometry">
        <rect>
         <x>720</x>
         <y>450</y>
         <width>80</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font/>
       </property>
       <property name="text">
        <string>注册</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_title">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>10</y>
         <width>590</width>
         <height>130</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="text">
        <string>物联网智慧农业系统</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>165</y>
         <width>321</width>
         <height>50</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_lusername">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>用户</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edit_lusername">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>ARM001</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QRadioButton" name="radio_offline">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>280</y>
         <width>130</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>离线模式</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radio_online">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>280</y>
         <width>110</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>在线模式</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_to_setserver">
       <property name="geometry">
        <rect>
         <x>690</x>
         <y>410</y>
         <width>110</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: transparent;</string>
       </property>
       <property name="text">
        <string>服务器设置</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_server">
      <widget class="QLabel" name="label_ip">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>179</y>
         <width>140</width>
         <height>33</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>服务器地址</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="edit_server_ip">
       <property name="geometry">
        <rect>
         <x>346</x>
         <y>181</y>
         <width>253</width>
         <height>29</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="text">
        <string>112.74.37.132</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_scancle">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>320</y>
         <width>100</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_sreturn">
       <property name="geometry">
        <rect>
         <x>680</x>
         <y>410</y>
         <width>80</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font/>
       </property>
       <property name="text">
        <string>返回</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="edit_server_port">
       <property name="geometry">
        <rect>
         <x>346</x>
         <y>240</y>
         <width>253</width>
         <height>29</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>55555</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_port">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>239</y>
         <width>140</width>
         <height>33</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>服务器端口</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_title_7">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>10</y>
         <width>590</width>
         <height>140</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="text">
        <string>物联网智慧农业系统</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_save_server">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>320</y>
         <width>100</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_register">
      <widget class="QLabel" name="label_title_2">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>0</y>
         <width>311</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>物联网智慧农业</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_register">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>200</y>
         <width>100</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>注册</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_rcancle">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>200</y>
         <width>100</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_tologin">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>240</y>
         <width>80</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font/>
       </property>
       <property name="text">
        <string>返回</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>142</y>
         <width>311</width>
         <height>36</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>确认密码</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edit_rpwd_2">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>100</y>
         <width>311</width>
         <height>36</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>密码</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edit_rpwd">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>60</y>
         <width>311</width>
         <height>36</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>用户</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edit_rusername">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edit_lusername</tabstop>
  <tabstop>edit_lpwd</tabstop>
  <tabstop>btn_login</tabstop>
  <tabstop>btn_cancle</tabstop>
  <tabstop>btn_toregister</tabstop>
  <tabstop>edit_rusername</tabstop>
  <tabstop>edit_rpwd</tabstop>
  <tabstop>edit_rpwd_2</tabstop>
  <tabstop>btn_register</tabstop>
  <tabstop>btn_rcancle</tabstop>
  <tabstop>btn_tologin</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
